manifests/ruby.yml

```diff
@@ -483,14 +494,36 @@ tests/:
     test_fingerprinting.py:
       Test_Fingerprinting_Endpoint: v2.15.0
       Test_Fingerprinting_Endpoint_Capability: missing_feature
-      Test_Fingerprinting_Endpoint_Preprocessor: missing_feature
+      Test_Fingerprinting_Endpoint_Preprocessor: v2.15.0-dev
       Test_Fingerprinting_Header_And_Network: v2.15.0
-      Test_Fingerprinting_Header_And_Network_Preprocessor: missing_feature
+      Test_Fingerprinting_Header_And_Network_Preprocessor: v2.15.0-dev
       Test_Fingerprinting_Header_Capability: missing_feature
       Test_Fingerprinting_Network_Capability: missing_feature
-      Test_Fingerprinting_Session: missing_feature
+      Test_Fingerprinting_Session:
+        "*": v2.15.0-dev
+        rack: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
+        sinatra14: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
+        sinatra20: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
+        sinatra21: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
+        sinatra22: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
+        sinatra30: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
+        sinatra31: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
+        sinatra32: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
+        sinatra40: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
+        uds-sinatra: irrelevant (We do not instrument authentication frameworks that work with sinatra or rack)
```

Comment:
If we do not instrument, I understand that we are not supporting it yet, so it should be `missing_feature`, because eventually it could be supported in the future.

Irrelevant is for tests that doesn't make sense, for example, some deleted feature, things that aren't implemented because they are not compatible with the framework.

Reply:
Ok, I will mark it as missing feature than